### DISCUSSION TITLE
nevim

### DISCUSSION THREAD

**Discussion by alenka1996** [community/community#31184] [Other Feature Feedback, Questions, & Ideas]
můžete mi někdo poradit? mam xiaomi a zničeho nic zamrznul, nereaguje na nic

**Comment by melvincwng** [comment]
@alenka1996 Hey there, I used Google translate to try to understand your question.

Unfortunately I'm not super familiar with hardware, specifically Xiaomi phone devices.

But if your Xiaomi phone isn't working, my suggestion would be to go to the nearest Xiaomi service center and get your phone fixed/replaced.

Alternatively, you can head over to the nearest phone repair centre to get it fixed.

Hope this answers your question, and have a good day ahead 😊
